### PR TITLE
docs(reference): grammar and conciseness changes

### PR DESCRIPTION
**Pull request by Fdawgs**
Reference counterpart to #3876

#### Checklist

- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs/Reference/Validation-and-Serialization.md [hunk 1]
```diff
@@ -165,7 +165,7 @@ of `'object'` and a `'properties'` object containing parameters) or a simpler
 variation in which the `type` and `properties` attributes are forgone and the
 parameters are listed at the top level (see the example below).
 
-> ℹ If you need to use the lastest version of Ajv (v8) you should read how to do
+> ℹ If you need to use the latest version of Ajv (v8) you should read how to do
 > it in the [`schemaController`](./Server.md#schema-controller) section. It is
 > explained the easier way to avoid to implement a custom validator.
```
Comment (Fdawgs) [inline docs/Reference/Validation-and-Serialization.md:170]: I have no idea what this sentence was trying to convey.

Comment (mcollina) [reply docs/Reference/Validation-and-Serialization.md:170]: remove?

Comment (Fdawgs) [reply docs/Reference/Validation-and-Serialization.md]: Removed in https://github.com/fastify/fastify/pull/3877/commits/482bf5782d4f04480c284d33bb96ac37fe74eb6f

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**